google/cloud/monitoring_v3/services/metric_service/client.py

```diff
@@ -166,14 +166,14 @@ def transport(self) -> MetricServiceTransport:
         return self._transport
 
     '''@staticmethod
-    def metric_descriptor_path(project: str,) -> str:
+    def metric_descriptor_path(project: str,metric_descriptor: str,) -> str:
         """Returns a fully-qualified metric_descriptor string."""
-        return "projects/{project}/metricDescriptors/{metric_descriptor=**}".format(project=project, )
+        return "projects/{project}/metricDescriptors/{metric_descriptor=**}".format(project=project, metric_descriptor=metric_descriptor, )
```

Comment:
I tried uncommenting the code but the unit tests fail as `**` is unsupported according to https://github.com/googleapis/gapic-generator-python/issues/701

```
_________________________________________________________________________________ test_metric_descriptor_path _________________________________________________________________________________

    def test_metric_descriptor_path():
        project = "squid"
        metric_descriptor = "clam"
>       expected = "projects/{project}/metricDescriptors/{metric_descriptor=**}".format(
            project=project, metric_descriptor=metric_descriptor,
        )
E       KeyError: 'metric_descriptor=**'

tests/unit/gapic/monitoring_v3/test_metric_service.py:3533: KeyError
______________________________________________________________________________ test_parse_metric_descriptor_path ______________________________________________________________________________

    def test_parse_metric_descriptor_path():
        expected = {
            "project": "whelk",
            "metric_descriptor": "octopus",
        }
>       path = MetricServiceClient.metric_descriptor_path(**expected)

tests/unit/gapic/monitoring_v3/test_metric_service.py:3545: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

project = 'whelk', metric_descriptor = 'octopus'

    @staticmethod
    def metric_descriptor_path(project: str, metric_descriptor: str,) -> str:
        """Returns a fully-qualified metric_descriptor string."""
>       return "projects/{project}/metricDescriptors/{metric_descriptor=**}".format(
            project=project, metric_descriptor=metric_descriptor,
        )
E       KeyError: 'metric_descriptor=**'

google/cloud/monitoring_v3/services/metric_service/client.py:171: KeyError
==============================================================================
```